web/sites/default/config/core.entity_view_display.node.park_facility.diff.yml

```diff
@@ -0,0 +1,213 @@
+uuid: b28fc8e0-7008-46bd-9c5f-3f18b4da8c69
+langcode: en
+status: false
```

Comment:
@kxwang, I am a little confused by all display modes that are in this PR. Do you know what triggered them to be created? There are no display modes configured for park facility when I look at the multidev for this one. Should we have default, teaser, featured, related, and search result configured so these other display modes don't appear?